javascript/javascript1/week4/main.js

```diff
@@ -0,0 +1,97 @@
+// Voice Assistant
+
+
+let personName = "";
+ const todoList = [];
+
+ function getReply(command) {
+    if (command === "" || typeof command !== "string") {
+        alert("Give some command Please!");
+    }
+    else if (command.includes("Hello my name is") && personName == "") {
+        personName = command.split(" ").pop();
+        return `Nice to meet you ${personName}`;
```

Comment:
Check `getReply("Hello my name is");` It will return `"Nice to meet you is"` and I would say it shouldn't 🙂